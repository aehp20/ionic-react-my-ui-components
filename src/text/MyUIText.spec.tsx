/**
 * @jest-environment jsdom
 */

import React from 'react'
import * as renderer from 'react-test-renderer'

import { MyUIText } from './MyUIText'

test("Component should show 'Name' text", () => {
  const component = renderer.create(<MyUIText>Name</MyUIText>)
  const tree = component.toJSON()

  expect(tree!.children).toMatchObject(['Name'])
  expect(tree).toMatchSnapshot()
})
