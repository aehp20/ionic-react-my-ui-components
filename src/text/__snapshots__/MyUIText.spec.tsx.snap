// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show 'Name' text 1`] = `
<ion-text>
  Name
</ion-text>
`;
